src/views/GeneralReception.vue

```diff
@@ -235,96 +284,163 @@ export default {
     barcodeCount: ({ barcodeArray }) => barcodeArray.length,
     presentRequestOptions: ({ requestOptions }) =>
       Object.fromEntries(Object.entries(requestOptions).filter(([, v]) => v)),
-    // DEPRECATE-DPL-877 once is enabled by default, remove this
-    version: () => {
-      // we only care if the feature is enabled or not, so we can just check for the presence of the feature
-      // if the feature flag can't be returned for any reason, we'll default to v1
-      const { data } = useSWRV(`${import.meta.env.VITE_TRACTION_BASE_URL}/flipper/api/actors/User`)
-      return data?.value?.features['dpl_877_reception_request']?.enabled ? 'v2' : 'v1'
+    //displayPrintOptions is used to decide whether print options should be displayed or not
+    displayPrintOptions: ({ source }) => source === 'SequencescapeTubes',
+    // printBarcodes is used to display the barcodes that will be printed
+    printBarcodes: ({ labwareData }) => Array.from(labwareData.foundBarcodes).join('\n'),
+    // printerOptions is used to display the printers that are available to print to
+    printerOptions() {
+      return this.$store.getters.printers.map((name) => ({
+        text: name,
+      }))
     },
+    // printEnabled is used to disable the print button if there are no barcodes to print
+    printEnabled: ({ printerName, printBarcodes }) => printerName && printBarcodes,
   },
   methods: {
-    importStarted({ message }) {
+    fetchStarted({ message }) {
       this.showModal(message)
     },
     clearModal() {
       this.modalState = defaultModal()
     },
+
     showModal(message) {
       this.modalState = { visible: true, message }
     },
-    importFailed({ message }) {
+    //Handler for deleting barcodes from the input field
```

Comment:
Not entirey sure what is going on here but you may be able to turn it into a map.

Reply:
This handler is designed for an edge case that addresses inconsistencies arising when the user enters barcodes and subsequently deletes them using the backspace or delete key. When the user inputs data, we fetch that information upon pressing the enter key, and our `labwareData` object keeps the necessary details from sequencescape fetch for printing or importing. 
However, if the user removes the barcode by pressing delete or backspace, the data stored in our state may become inconsistent with what is present in the input barcode field. Therefore, it is essential to delete all information corresponding to the deleted input barcode. Failing to do so could result in printing or importing information related to barcodes that are no longer in the input barcode field.  
Hope it make sense.

As we are keeping three different array fields  in `labwareData` for inputBarcodes, foundBarcdes(from SS) and attributes (again from SS for import) I thought forEach is better to make changes for each of them. 

Please let me know your thoughts on this.

Reply:
The alternative solution to completely eliminate this issue and this handler is to inform the user that, after making any changes in input barcode field( whether it is delete or any edit), they should press 'Enter'. This action will ensure that the labwareData state aligns consistently with the barcodes in the input field.

Reply:
Yes sure. Do you have some time tomorrow after 10.30 , for a quick call?